packages/webview_flutter_lwe/example/lib/main.dart

```diff
@@ -108,15 +114,30 @@ class WebViewExample extends StatefulWidget {
 }
 
 class _WebViewExampleState extends State<WebViewExample> {
-  final WebViewController _controller = WebViewController();
+  late final WebViewController _controller;
 
   @override
   void initState() {
     super.initState();
 
-    _controller
+    // #docregion platform_features
+    late final PlatformWebViewControllerCreationParams params;
+    if (WebViewPlatform.instance is WebKitWebViewPlatform) {
+      params = WebKitWebViewControllerCreationParams(
+        allowsInlineMediaPlayback: true,
+        mediaTypesRequiringUserAction: const <PlaybackMediaTypes>{},
+      );
+    } else {
+      params = const PlatformWebViewControllerCreationParams();
+    }
+
+    final WebViewController controller =
+        WebViewController.fromPlatformCreationParams(params);
+    // #enddocregion platform_features
+
+    controller
       ..setJavaScriptMode(JavaScriptMode.unrestricted)
-      ..setBackgroundColor(const Color(0x00000000))
+      ..setBackgroundColor(const Color(0x80000000))
```

Comment:
Please keep the code related of setBackgroundColor() in the example code of webview_flutter.
(I think it would be better to handle the issue of setBackgroundColor() not working after loadRequest() on a specific site separately. Or update it together in this PR.)